src/indexers/vault-harvests-indexer.ts

```diff
@@ -21,7 +21,10 @@ export async function updateVaultHarvests() {
         console.log(`[${name}]: Discovered ${yieldEvents.length} yield events`);
 
         if (yieldEvents.length === 0) {
-          vault.lastHarvestIndexedBlock = currentBlock;
+          vault.lastHarvestIndexedBlock = Math.min(
```

Comment:
this caused us to fully fast forward, 
here we will use the lower value so if the presumed scan resulted in nothing, we either skip blocks, or fast forward to current